### PR TITLE
Create el_gr.json for Mod Menu Helper

### PR DESCRIPTION
**Pull request by RaptaG**
:D

### CODE REVIEW

File: Mod Menu Helper/assets/modmenu/lang/el_gr.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+  "modmenu.dropInfo.line1": "Οι τροποποιήσεις του FO έχουν  §9✎§r ή §8✎§r",
+  "modmenu.dropInfo.line2": "οι εναλλακτικές έχουν §c*§r, οι υπόλοιπες δεν υποστηρίζονται",
+  "modmenu.summaryTranslation.slight-gui-modifications": "Animations, έκδοση πακέτου στο μενού\n§9✎§8 Animations, χρώματα, εμφάνηση FPS, ...",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/el_gr.json:4]: Shouldn't "animations" be "κινούμενα σχέδια"?

File: Mod Menu Helper/assets/modmenu/lang/el_gr.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  "modmenu.summaryTranslation.advancementinfo": "Μεγαλύτερη, εξυπνότερη οθόνη επιτευγμάτων\n§9✎§8 Χρώματα, μέγεθος του πεδίου πληροφοριών, ...",
+  "modmenu.summaryTranslation.advancements-enlarger": "§c*§r Μεγαλύτερη οθόνη επιτευγμάτων\n§8✎ (χωρίς επιλογές)",
+  "modmenu.summaryTranslation.animatica": "Τα κινούμενα είδη του Optifine\n§8✎ (μόνο εξιδικευμένες επιλογές)",
+  "modmenu.summaryTranslation.antighost": "Διορθώνει τους αόρατους κύβους με το /ghost\n§9✎§8 Πλήκτρο",
+  "modmenu.summaryTranslation.betterbeds": "§c*§r Γρηγορότερα κρεβάτια\n§8✎ (χωρίς επιλογές)",
+  "modmenu.summaryTranslation.bettermounthud": "Εμφάνηση φαγητού-εμπιρείας πάνω σε άλογο\n§8✎ (χωρίς επιλογές)",
+  "modmenu.summaryTranslation.bobby": "Ανοιχτά φορτώμενα κομμάτια server\n§9✎§8 Κύβοι-οντότητες, παραμόνη cache μνήμης, ...",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/el_gr.json:11]: Cache? server?

File: Mod Menu Helper/assets/modmenu/lang/el_gr.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  "modmenu.summaryTranslation.fabrishot": "Στιγμιότυπα οθόνης μεγάλης ανάλησης\n§9✎§8 Προσπέλαση F2, ανάλυση, πλήκτρο, ...",
+  "modmenu.summaryTranslation.farsight": "§c*§r Κραταέι τα φορτωμένα κομμάτια του κόσμου σε εξυπηρετές\n§8✎ (χωρίς επιλογές)",
+  "modmenu.summaryTranslation.fastchest": "§c*§r Γρηγορότερα σεντούκια\n§9✎§8 Άνοιγμα/Κλέισιμο",
+  "modmenu.summaryTranslation.fastopenlinksandfolders": "Όχι πάγωμα ανοίγοντας σύνδεσμους - φάκελους\n§8✎ (χωρίς επιλογές)",
+  "modmenu.summaryTranslation.ferritecore": "Μειώση χρήσης RAM μνήμης\n§8✎ (μόνο εξιδικευμένες επιλογές)",
+  "modmenu.summaryTranslation.forgetmechunk": "Μείωση κολλήματων μέσα σε κόσμους\n§8✎ (χωρίς επιλογές)",
+  "modmenu.summaryTranslation.iris": "Οι σκιάσεις του Optifine\n§9✎§8 Άνοιγμα/Κλέισιμο, επιλογή σκιάσεων, πλήκτρο ...",
+  "modmenu.summaryTranslation.itemmodelfix": "Διόρθωση διαφανών γραμμών στα αντικείμενα\n§9✎§8 Μέθοδος",
+  "modmenu.summaryTranslation.lambdabettergrass": "Το \"βελτιωμένο γρασίδι\" του OptiFine\n§9✎§8 Ποιότητα, χιονισμένο γρασίδι, πακέτο πόρων",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/el_gr.json:32]: To?

**Review by Madis0 — CHANGES_REQUESTED** [review]

File: Mod Menu Helper/assets/modmenu/lang/el_gr.json (hunk 3 above)
Comment (RaptaG) [reply Mod Menu Helper/assets/modmenu/lang/el_gr.json:32]: The greek "το"

File: Mod Menu Helper/assets/modmenu/lang/el_gr.json (hunk 2 above)
Comment (RaptaG) [reply Mod Menu Helper/assets/modmenu/lang/el_gr.json:11]: cache goes for προσωρινή μνήμη, but it captures too much space, and I had to find a way to make it fit in

Comment (RaptaG) [reply Mod Menu Helper/assets/modmenu/lang/el_gr.json:11]: for server too

File: Mod Menu Helper/assets/modmenu/lang/el_gr.json (hunk 1 above)
Comment (RaptaG) [reply Mod Menu Helper/assets/modmenu/lang/el_gr.json:4]: not the meaning that animation here means

**Pull request merged**